libs/src/api/user.js

```diff
@@ -234,9 +230,7 @@ class Users extends Base {
       // Autoselect a new replica set and update the metadata object with new content node endpoints
       phase = phases.AUTOSELECT_CONTENT_NODES
       const response = await this.ServiceProvider.autoSelectCreatorNodes({
-        performSyncCheck: false,
-        whitelist: passList,
```

Comment:
what's the reason for removing these params?

Reply:
they're provided as ctor args to the creator node config class, so they would be redundant here! did that to just to consolidate the logic around selection & fix the existing dapp bug where we don't call autoSelect with an object for args